### PR TITLE
Boyardee updates 2

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request
Another update to the Boyardee class 

- Meat fridge has extra meat
- Added a nutrivend
- Replaced the two empty monitors in the bridge with a civilian mod console and a power monitor
- Added a clothesmate
- Added a totally normal pet chicken and a totally normal strawbed for her
- Moved ice-cream vat to the cold room (why would you store your icecream right next to the engines?)
- PAI card moved from ontop of the helm to the bar
- Renamed external airlocks to make setting up the AAC easier
- AAC starts unlocked
- Added a sink inside of botany
- Added a completely normal chicken

## Why It's Good For The Game
I love but boyardee but it needs a little love, this update mostly makes it easier to produce food 

## Changelog
:cl:
add: Small tweaks around the Boyardee
/:cl:

### CODE REVIEW

**Comment by Apogee-dev** [comment]
Please update the Boyardee’s config file with ship name lists, outfits, and so on while you’re at it. Most of the older ships still don’t have those defined. Check one of the more up-to-date ships like the Carina if you need a reference.

**Review by triplezeta — APPROVED** [review]
ok!

**Comment by MitztheKat** [comment]
fucking spess chicken

File: _maps/shuttles/shiptest/bar_ship.dmm [hunk 1]
```diff
@@ -1093,6 +1116,16 @@
...
 	},
+/mob/living/simple_animal/chicken{
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0);
+	desc = "A regular chicken, nothing weird about this one .";
+	name = "Cluckens"
+	},
+/obj/structure/bed/nest{
+	color = "#FFFF00";
+	desc = "It's a pile of soft, loose hay shaped like a nest.";
+	name = "Chicken Nest"
```
Comment (Fikou) [inline _maps/shuttles/shiptest/bar_ship.dmm]: object names should be lower case

Comment (Fikou) [reply _maps/shuttles/shiptest/bar_ship.dmm]:
```suggestion
	name = "chicken nest"
```

**Pull request merged**